src/components/custom-field-input-single-choice/custom-field-input-single-choice.css

```diff
@@ -1,28 +1,3 @@
-@import '../../styles/colors-v2.css';
-@import '../../styles/spacing.css';
-
 .container {
```

Comment:
Do we still need this around for single choice? Looks like Select implements the same rule on its root node

Reply:
Good point, with that the whole style file can be removed.